modules/ravestate_nlp/__init__.py

```diff
@@ -62,6 +70,14 @@ def nlp_preprocess(ctx):
     ctx["nlp:roboy"] = nlp_roboy
     logger.info(f"[NLP:roboy]: {nlp_roboy}")
 
+    nlp_yesno = nlp_doc._.yesno
+    ctx["nlp:yesno"] = nlp_yesno
+    logger.info(f"[NLP:yesno]: {nlp_yesno}")
+
+    if nlp_triples[0].match_either_lemma(pred={"play"}, obj={"game"}):
```

Comment:
Why is this a property and not an nlp:intent-play signal or so?

Reply:
Changed it :sheep: